spec/interaction-spec.js

```diff
@@ -34,6 +45,82 @@ describe('c3 chart interaction', function () {
                 });
             });
 
+            describe('mouseover', function () {
+                beforeAll(function () {
+                    args = {
+                        data: {
+                            columns: [
+                                ['data1', 30, 200, 100, 400, -150, 250],
+                                ['data2', 50, 20, 10, 40, 15, 25],
+                                ['data3', -150, 120, 110, 140, 115, 125]
+                            ],
+                            type: 'bar',
+                            onmouseout: function() {
+                                window.mouseoutCounter += 1;
+                            },
+                            onmouseover: function() {
+                                window.mouseoverCounter += 1;
+                            }
+                        },
+                        axis: {
+                            rotated: false
+                        }
+                    };
+                });
+
+                beforeEach(function() {
+                    window.mouseoverCounter = 0;
```

Comment:
Do these variables need to be global? How about declaring them with `let` under `describe('mouseover'` scope?